scoring/scoring.py

```diff
@@ -41,11 +49,102 @@
     'wer',
     'l1_loss',
 ]
+
 MAX_EVAL_METRICS = ['average_precision', 'ssim', 'accuracy', 'bleu_score']
+WORKLOADS = {
```

Comment:
we should be able to just import workloads and use the var/import helper from there, right?